packages/api/src/commands/install.ts

```diff
@@ -103,7 +103,7 @@ cmd
       // logger(`It looks like you already have this API installed. Would you like to update it?`);
     }
 
-    let spinner = ora('Fetching your API definition').start();
+    let spinner = ora({ text: 'Fetching your API definition', ...oraOptions() }).start();
```

Comment:
i created a shared `oraOptions` config (similar to how we do in `rdme`) so we don't log `ora` spinners in tests.